CMakeLists.txt

```diff
@@ -97,7 +97,12 @@ function(build_ngraph)
     elseif(WIN32)
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4308 /wd4146 /wd4703 /wd4244 /wd4819")
     endif()
-    
+
+    # WA for GCC 7.5 "PYBIND11_NOINLINE inline" warning
+    if (UNIX)
+        ie_add_compiler_flags(-Wno-error=attributes)
+    endif()
```

Comment:
can we move this directly to pyngraph target? E.g. use `target_compile_options`

Reply:
Sure, done!